src/funding/paypal/config.jsx

```diff
@@ -45,18 +45,21 @@ export function getPayPalConfig(): FundingSourceConfig {
 
     labelText: ({ content, label, period }) => {
       let text = `${FUNDING_BRAND_LABEL.PAYPAL}`;
+
       if (content && label === BUTTON_LABEL.INSTALLMENT) {
         if (period) {
-          text = content["label.installment.withPeriod"].replace(
-            "{period}",
-            String(period)
-          );
+          const rawLabel = content["label.installment.withPeriod"];
+
+          if (typeof rawLabel === "string") {
+            text = rawLabel.replace("{period}", String(period));
+          }
```

Comment:
@wsbrunson is it worth adding any logging when this type check fails?

Reply:
I don't think so, it could be totally normal not to have this specific string